src/fides/api/models/privacy_experience.py

```diff
@@ -141,67 +392,158 @@ def dry_update(self, *, data: dict[str, Any]) -> PrivacyExperienceConfig:
-
-    @classmethod
-    def get_default_config(
-        cls, db: Session, component: ComponentType
-    ) -> Optional[PrivacyExperienceConfig]:
-        """Load the first default config of a given component type"""
         return (
-            db.query(PrivacyExperienceConfig)
-            .filter(PrivacyExperienceConfig.component == component)
-            .filter(PrivacyExperienceConfig.is_default.is_(True))
-            .first()
+            self.experience_config_history.id
+            if self.experience_config_history
+            else None
         )
 
 
-class PrivacyExperienceConfigHistory(ExperienceConfigBase, Base):
-    """Experience Config History - stores the history of how the config has changed over time"""
+class PrivacyExperienceConfigHistory(
+    ExperienceTranslationBase, PrivacyExperienceConfigBase, Base
+):
+    """Experience Config History table for auditing purposes.
 
-    experience_config_id = Column(
-        String, ForeignKey(PrivacyExperienceConfig.id_field_path), nullable=False
+    When an Experience Config and/or an Experience Translation is modified, a new version is
+    created here that stores a snapshot of these records combined.  Consent reporting can contain
+    an id to this resource which preserves the details of the Experience viewed by the end user.
+    """
+
+    custom_asset_id = Column(String, ForeignKey(CustomAsset.id_field_path))
+    origin = Column(String, ForeignKey(ExperienceConfigTemplate.id_field_path))
+
+    translation_id = Column(
+        String, ForeignKey(ExperienceTranslation.id_field_path, ondelete="SET NULL")
     )
 
+    version = Column(Float, nullable=False, default=1.0)
+
+    banner_enabled = Column(String(), index=True)  # TODO pending removal
+
+    experience_config_id = Column(
+        String, ForeignKey(PrivacyExperienceConfig.id_field_path), nullable=True
+    )  # TODO slated for removal after data migration
+
 
 class PrivacyExperience(Base):
-    """Stores Privacy Experiences for a given just a single region.  The Experience describes how to surface
-    multiple Privacy Notices or TCF content to the end user in a given region.
+    """Privacy Experiences connect a region to a shared PrivacyExperienceConfig.
 
-    There can only be one component per region.
+    This resource should be queried for an end-user in a specific region, and the configuration
+    will be loaded from the shared Privacy Experience Config.
     """
 
-    component = Column(EnumColumn(ComponentType), nullable=False)
     region = Column(EnumColumn(PrivacyNoticeRegion), nullable=False, index=True)
 
     experience_config_id = Column(
         String,
         ForeignKey(PrivacyExperienceConfig.id_field_path),
-        nullable=True,
+        nullable=True,  # Needs an ExperienceConfig to be valid though
         index=True,
     )
 
-    # There can only be one component per region
-    __table_args__ = (UniqueConstraint("region", "component", name="region_component"),)
-
```

Comment:
We used to have a unique constraint here on region and component. The component field has been removed, and now PrivacyExperiences are purely 1) how regions are linked to a PrivacyExperienceConfig and 2) the primary artifact queried by the end user because it is region-specific.